build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -13,18 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+// @ts-nocheck
```

Comment:
why? is this a particularly bad one?

Reply:
Yeah, the `ControllerPromise` type is annotated as though it extends `Promise` and treated as such but does not actually extend `Promise`. I was hoping to address that as a functional change in a separate PR.

Reply:
Perhaps this file too should be excluded via `build-system/tsconfig.json`?

Reply:
See https://github.com/ampproject/amphtml/pull/33166#discussion_r612793826. This file "only" is responsible for 9 errors but they are pretty hard (maybe impossible?) to fix in a non functional way. I'll remove this ts-nocheck for now for the same reasons stated in the linked comment and will address this file in its own PR.